course-info/yarn.lock

```diff
@@ -1,656 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This `yarn.lock` needs to be removed since it prevents `course-info` from being recognized as a workspace.